### DISCUSSION TITLE
"Library dashboard"

### DISCUSSION THREAD

**Discussion by radimkohout** [grafana/grafana#46847] [Dashboards, Panels & Visualizations]
Hello,
it would be nice to have a sort of library dashboards - working like library graphs but with dashboards.
It should have the function, that you could few graph queries on the dashboard, and it would change on all linked dashboards. 
Another option would be to link all graphs in the dashboard.

It's really annoying doing it one by one for like 20 graphs.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
